### PR TITLE
BG-1366: Donation widget accent color

### PR DESCRIPTION
**Pull request by ap-justin**
## Explanation of the solution
* add color controls
image

* adjust styling to use custom accent
image

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/components/donation/Steps/Submit/DAFCheckout/ManualDonation.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function ManualDonation(props: DafCheckoutStep) {
         To complete this donation, please email your provider with the following
         information:
       </p>
-      <div className="grid rounded bg-gray-l5 dark:bg-navy-d3 p-3 text-sm leading-relaxed my-4">
+      <div className="grid rounded bg-gray-l4 dark:bg-navy-d3 p-3 text-sm leading-relaxed my-4">
```
Comment (ap-justin) [inline src/components/donation/Steps/Submit/DAFCheckout/ManualDonation.tsx:17]: `bg-gray-l5` has blueish hue, `-l4` is more neutral and would blend to any accent

File: src/components/donation/Steps/Submit/DAFCheckout/ManualDonation.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ export default function ManualDonation(props: DafCheckoutStep) {
           profileUrl,
           props.details.amount
         )}
-        className="mt-1.5 btn btn-blue rounded-full px-4 py-2 justify-self-end text-xs font-normal"
+        className="mt-1.5 btn btn-blue bg-[--accent-primary] enabled:hover:bg-[--accent-primary] rounded-full px-4 py-2 justify-self-end text-xs font-normal"
```
Comment (ap-justin) [inline src/components/donation/Steps/Submit/DAFCheckout/ManualDonation.tsx:61]: hover color is set to be the same as base color as its another problem to determine lighter (simply decreasing opacity is suboptimal) color base on a given accent

File: src/components/donation/Steps/Submit/StripeCheckout/StripeCheckout.tsx [hunk 3]
```diff
@@ -64,8 +64,22 @@ export default function StripeCheckout(props: StripeCheckoutStep) {
...
             options={{
+              fonts: [
+                {
+                  family: "DM Sans",
+                  cssSrc: "https://fonts.googleapis.com/css2?family=Quicksand",
+                },
+              ],
               clientSecret,
-              appearance: { theme: "stripe" },
+              appearance: {
```
Comment (ap-justin) [inline src/components/donation/Steps/Submit/StripeCheckout/StripeCheckout.tsx:74]: styling tweaks for stripe
image

**Review by SovereignAndrey — APPROVED** [review]
This looks great!! :heart:

**Pull request merged**